.github/workflows/close.yaml

```
@@ -0,0 +1,51 @@
+name: "Close"
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
```

This should be replaced with

```suggestion
  pull_request_target:
```

to allow closing pull requests from forks.